command/push.go

```diff
@@ -95,6 +97,12 @@ func doPush(ctx *cli.Context) {
 		}
 	}
 
+	if bigImageUrl != "" {
+		msg.Extras["client::notification"] = map[string]interface{}{
```

Comment:
With this, it won't be possible to supply both --clickUrl and --bigImageUrl together, as bigImageUrl currently overrides clickUrl.

Reply:
You’re right. Fixed in the new version I just forced push (tested and verified)